static/src/stylesheets/module/facia-garnett/_container.scss

```diff
@@ -132,6 +132,10 @@ $header-image-size-desktop: 100px;
     padding-bottom: $gs-baseline/3;
     color: $brightness-7;
 
+    h2 {
+        font-size: 1.37rem; //Quick fix for a Glabs title issue, check out PR #24458 for the details
```

Comment:
Sidenote. Comments in css should use the /* my comment */ pattern. `//` can bite back in painful ways.

See: https://developer.mozilla.org/en-US/docs/Web/CSS/Comments

Reply:
Interesting, // is used quite liberally in this file already lol

Reply:
https://sass-lang.com/documentation/syntax/comments#:~:text=Comments%20in%20SCSS%20work%20similarly,may%20as%20well%20not%20exist.

Reply:
🙃 it ain't valid css...

I suspect we _get away_ with it because of loose conventions (or maybe build steps?) but for DCR I've seen this blow up pages on the site in spectacular ways.

Reply:
> in Sass

Oh, lol, it ain't css at all.